public/index.html

```diff
@@ -22,6 +22,9 @@
     	<script src="/scripts/imageGridScript.js"></script>
     	<script src="/scripts/timeline.js?version=1005"></script>
     	<script src="/scripts/waitforimages.js"></script>
+	<script src="/scripts/gifshot.js"></script>
```

Comment:
```suggestion
```

Since production-friendly minified `gifshot.min.js` is included, including debugging-friendly version too is redundant.